src/main/java/com/atlassian/db/replica/internal/aurora/AuroraReplicaNode.java

```diff
@@ -32,20 +36,24 @@ public String getId() {
     public Supplier<Connection> getConnectionSupplier() {
         return () -> {
             try {
+                Connection connection = getAuroraConnectionSupplier().get();
                 return replicaNode.mark(
```

Comment:
I wonder if we can stop marking the connection if we'll have API to acquire connection to specific replica

Reply:
replicaNode is not being used anywhere else in this class so marking the node does virtually nothing. it could be simply removed from the code with no effect on this class behaviour